### PR TITLE
Feature/mobile menu

### PR DESCRIPTION
**Pull request by Shahid-Bagwan**
- add mobile menu.
- add style for the menu.
- change the height o background images

### CODE REVIEW

**Review by Efiamotu-1 — COMMENTED** [review]
Hi @Shahid-Bagwan , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request
 

You have submitted a project, that doesn't have all linters setup. currently you do not have linters to check your javascript code visit this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to properly setup linters and resubmit :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    title.classList.toggle('test');
+    nav.classList.toggle('navbar');
+    nav.classList.toggle('navbar1');
+    i.classList.toggle('fas');
+    i.classList.toggle('fa-bars');
+    i.classList.toggle('fa-xs');
+    i.classList.toggle('fa-solid');
+    i.classList.toggle('fa-xmark');
+  }
+  navToggle.addEventListener('click', () => toggle());
```
Comment (zieeco) [inline index.js]: - Hey, according to the requirement, if a user clicks on any of the menu items, it should take the user to that specific part of the webpage. However, this has not been implemented. please, let's fix this.
- Also, on clicking the different links, the menu should disappear. please, fix this also.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Afsheen96,




Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Mobile menu is almost done.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions


- N/A
 
**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Afsheen96** [comment]
Which repository is this???

File: index.js (hunk 1 above)
Comment (Afsheen96) [reply index.js]: Sorry sir i am confused as i am not the collaborator of this reop this is not my project so why did you mention me 🤔🤔🤔 @zieeco

Comment (Shahid-Bagwan) [reply index.js]: done
!

**Review by usorfaitheloho — APPROVED** [review]
Hi @Shahid-Bagwan  👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**